maintenance/countBlogWatchers.php

```diff
@@ -0,0 +1,82 @@
+<?php
+require_once __DIR__ . '/Maintenance.php';
+
+class CountBlogWatchers extends Maintenance {
+
+	public function __construct() {
+		parent::__construct();
+		$this->addOption( 'blog', 'The blog watchlist to recount', true, true );
+		$this->mDescription = 'Count all real subscribers to the watchlist of a seelected blog';
+	}
+
+	public function execute() {
+		if ( !$this->hasOption( 'blog' ) ) {
+			$this->error( 'A blog is required' );
+		}
+
+		$blog = $this->getOption( 'blog' );
+
+		$dbWikia = wfGetDB( DB_SLAVE, [], 'wikia' );
+		$dbWikicities = wfGetDB( DB_SLAVE, [], 'wikicities' );
+		$dbUserPreferences = wfGetDB( DB_SLAVE, [], 'user_preferences' );
+
+		// take all user ids from watchlist
+		$watchlistSQL = '
+			SELECT DISTINCT(wl_user) user_id
+			FROM watchlist 
+			WHERE 
+				wl_title LIKE "' . trim( $blog ) . '"
+			AND wl_namespace = 502;
+		';
+
+		$res = $dbWikia->query( $watchlistSQL, 'CountBlogWatchers::execute' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "CountBlogWatchers::execute" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)